packages/navigation-menu/src/lib/navigation-menu/components/sub-menu-item/sub-menu-item.component.html

```diff
@@ -0,0 +1,15 @@
+<li [ngClass]="{'has-sub-menu' : (!!subMenu), 'docked' : (state$ | async)?.docked}" #inclusedContent>
+	<a *ngIf="href || routerLink" [href]="href ? href : null" [routerLink]="routerLink ? routerLink : null">
+		<ng-container *ngTemplateOutlet="link"></ng-container>
+		<span *ngIf="!!subMenu || !!forceShowArrow" class='fa fa-angle-down'></span>
```

Comment:
Please use consistent double quotes in your HTML.

Reply:
I get that you focus on consistency, but since these files are processed by webpack, it doesn't matter at all. But I'll change it for you ;)